### PR TITLE
feat: consume package from @dcl/explorer instead of @dcl/kernel and @…

### PR DESCRIPTION
**Pull request by kuruk-mm**
…dcl/unity-renderer

### CODE REVIEW

File: scripts/copy_cdn.js [hunk 1]
```diff
@@ -34,7 +34,6 @@ function copyPackage(packageName, envValueToCheck) {
   console.log('\n')
 }
 
-copyPackage('@dcl/unity-renderer', process.env.REACT_APP_RENDERER_BASE_URL)
-copyPackage('@dcl/kernel', process.env.REACT_APP_KERNEL_BASE_URL)
+copyPackage('@dcl/unity-renderer', process.env.REACT_APP_EXPLORER_BASE_URL)
```
Comment (menduz) [inline scripts/copy_cdn.js:37]:
```suggestion
copyPackage('@dcl/explorer', process.env.REACT_APP_EXPLORER_BASE_URL)
```

File: scripts/hash_generator.js [hunk 2]
```diff
@@ -12,13 +12,10 @@ if (fs.existsSync('.env')) {
 const packageJson = JSON.parse(fs.readFileSync('./package.json').toString())
 const publicPackageJson = JSON.parse(fs.readFileSync('./public/package.json').toString())
 
-const kernelVersion = JSON.parse(fs.readFileSync(require.resolve('@dcl/kernel/package.json'))).version
-const rendererVersion = JSON.parse(fs.readFileSync(require.resolve('@dcl/unity-renderer/package.json'))).version
+const explorerVersion = JSON.parse(fs.readFileSync(require.resolve('@dcl/unity-renderer/package.json'))).version
```
Comment (menduz) [inline scripts/hash_generator.js:15]:
```suggestion
const explorerVersion = JSON.parse(fs.readFileSync(require.resolve('@dcl/explorer/package.json'))).version
```

**Review by menduz — APPROVED** [review]
Lovely

**Comment by kuruk-mm** [comment]
This PR must be merged after we have a stable release of the `@dcl/explorer`.

**Review by menduz — APPROVED** [review]

**Pull request merged**